auv_sim/auv_sim_bridge/include/auv_sim_bridge/simulation_mock_ros.h

```diff
@@ -81,24 +120,32 @@ class SimulationMockROS {
     depth_pub_.publish(depth_msg);
   }
 
-  void dvlCallback(const uuv_sensor_ros_plugins_msgs::DVL &msg) {
+  void altimeterCallback(const sensor_msgs::Range &msg) {
     std_msgs::Float32 altitude_msg;
-    geometry_msgs::Twist velocity_msg;
-    geometry_msgs::TwistWithCovarianceStamped velocity_stamped_msg;
-    // Dvl altitude
-    altitude_msg.data = msg.altitude;
+
+    altitude_msg.data = msg.range;
     altitude_pub_.publish(altitude_msg);
 
+    latest_altitude_ = msg.range;
+  }
+
+  void dvlCallback(const nav_msgs::Odometry &msg) {
+    geometry_msgs::Twist velocity_raw_msg;
+    std_msgs::Bool is_valid_msg;
+
+    if (latest_altitude_ > 0.3 && dvl_enabled_) {
```

Comment:
Lets make 0.3 either a rosparam or a constant st the top of the file. A constant would be sufficient.